### ISSUE TITLE
False positive in conflicts report

### ISSUE THREAD

**Issue by lionel-** [r-lib/pkgload#125]
I'm seeing:

```r
── Conflicts ──────────────────────────────────────────────────────── errors conflicts ──
✖ df() masks errors::df()
```

However the `df` object in the global environment is a data frame, not a function. Maybe this shouldn't show up as a conflict?

**Comment by jimhester** [comment]
yeah, I think that makes sense.

**Comment by lionel-** [comment]
Duplicate of #143, closed by #144.
